pymemcache/client/base.py

```diff
@@ -304,6 +305,9 @@ def set(self, key, value, expire=0, noreply=None):
                   from the cache, or zero for no expiry (the default).
           noreply: optional bool, True to not wait for the reply (defaults to
                    self.default_noreply).
+          flag: memcached operation flag
```

Comment:
(Let's swap the order of `flags` and `encoding` here, too.

We should also provide a little more document borrowed from the protocol:

https://github.com/memcached/memcached/blob/78eb7701e0823643d693c1a7a6fd8a0c75db74d8/doc/protocol.txt#L175-L181

Something like:

```
flags: int, arbitrary bit field used for server-specific flags
```